templates/collection_view_screen/app/stylesheets/name_cell_stylesheet.rb

```diff
@@ -0,0 +1,13 @@
+module <%= @name_camel_case %>CellStylesheet
+  def cell_size
+    {w: 96, h: 96}
+  end
+
+  def <%= @name %>_cell(st)
+    st.frame = cell_size
+    st.background_color = color.random
```

Comment:
I'm very unlikely to use a color.random background and have to change it every time - but yeah I get you.